spec/controllers/users_controller_spec.rb

```diff
@@ -55,8 +55,8 @@
       @params[:id] = user.id
     end
 
-    it_should_allow_admin_only do
-      expect(assigns[:user]).to eq(user)
+    context "when create_users is active", feature_setting: { create_users: true } do
```

Comment:
I would probably pull this flag up to the `describe "GET #edit"` line, but it's not a big deal here. But in general, if having the feature off means there is no behavior, it's usually safe to not do the inverse.